### ISSUE TITLE
 make CFLAGS="-Wmissing-declarations" fails

### ISSUE THREAD

**Issue by tmakatos** [nutanix/libvfio-user#342]
```
 make CFLAGS="-Wmissing-declarations"
...
[ 15%] Building C object lib/CMakeFiles/dma.dir/dma.c.o
/root/src/libvfio-user/lib/dma.c:94:1: error: no previous declaration for ‘_dma_controller_do_remove_region’ [-Werror=missing-declarations]
 _dma_controller_do_remove_region(dma_controller_t *dma,
 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/root/src/libvfio-user/lib/dma.c:420:9: error: no previous declaration for ‘_get_bitmap_size’ [-Werror=missing-declarations]
 ssize_t _get_bitmap_size(size_t region_size, size_t pgsize)
```

**Comment by jlevon** [comment]
oh, definitely worth adding!

**Comment by jlevon** [comment]
Looks like this wasn't actually fixed

**Comment by jlevon** [comment]
fixed by #399
